### ISSUE TITLE
Floating titles for polylines

### ISSUE THREAD

**Issue by sballe73** [androidseb/mapmarker#20]
add an option to display its length, for example at the middle of the first two points of the line.

**Comment by androidseb** [comment]
Today, the length of a line is 3 clicks away, you need to:

* Click the line
* Click the "Edit" button
* Scroll down to the bottom to see the value

If you edit your POI information to put the length in the title or the description, then it becomes 1 click away (clicking on the line, you can see its length in the description).

What if paths had the "floating title" functionality, would that work for what you're trying to do?

**Comment by sballe73** [comment]
right, I just used the insert into title and it does enough for me. Having the floating title option would be a bonus :)

by the way thanks for your app, it is really helping me for my task this year.

**Comment by androidseb** [comment]
Thanks, I'll rename this issue to be about floating titles for polylines then!
